digiroad2-oracle/src/main/resources/db/migration/V210__add_table_dataset.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE DATASET (
+ DATASET_ID CHAR(36) PRIMARY KEY,
+ GEOJSON CLOB NOT NULL,
+ ROADLINKS CLOB NOT NULL,
+ RECEIVED_DATE TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL,
+ PROCESSED_DATE TIMESTAMP,
+ STATUS NUMBER
+);
+
+CREATE TABLE FEATURE (
+ FEATURE_ID NUMBER,
+ DATASET_ID CHAR(36) REFERENCES DATASET(DATASET_ID),
+ STATUS VARCHAR2(4000),
```

Comment:
Why is one status a number and the other one a big varchar?

Reply:
The idea behind that was that the dataset can only have one status simply or its wrong(because of the features) or it is right. The features can have various status depending on what is wrong inside the feature.